python/fapolicy_analyzer/ui/trust_file_list.py

```diff
@@ -106,8 +107,39 @@ def on_addBtn_clicked(self, *args):
         )
         fcd.set_select_multiple(True)
         response = fcd.run()
+        fcd.hide()
         if response == Gtk.ResponseType.OK:
             files = [f for f in fcd.get_filenames() if path.isfile(f)]
+
+            # -- Filter to address fapolicyd embeded whitspace in path issue
+            #     Current fapolicyd VT.B.D. When fixed remove this block
+            #
+            # Detect and remove file paths w/embedded spaces. Alert user w/dlg
+            print("Filtering out paths with embedded whitespace")
+            listTmp = []
+            for e in files:
+                if not re.search(r"\s", e):
+                    listTmp.append(e)
+                else:
+                    dlgWhitespaceInfo = Gtk.MessageDialog(
+                        transient_for=self.trustFileList.get_toplevel(),
+                        flags=0,
+                        message_type=Gtk.MessageType.INFO,
+                        buttons=Gtk.ButtonsType.OK,
+                        text="File path contains embedded whitespace.",
+                    )
+                    dlgWhitespaceInfo.format_secondary_text(
+                        f"{e}, contains whitespace.\nfapolicyd "
+                        + "currently does not support paths containing spaces."
+                        + "\n\nPath will not be added to the Trusted Files "
+                        + "List. (fapolicyd: VTBD)"
+                    )
+                    dlgWhitespaceInfo.run()
+                    dlgWhitespaceInfo.destroy()
+            files = listTmp
```

Comment:
My only comment on this is that maybe we compile a list of files we removed and give one warning message.  If they select 6 files with spaces it would be rather annoying to get the popup 6 times in a row.  You could do something like this with list comprehensions
```python
filtered = [f for f in files if not re.search(r"\s", f)]
diff = [f for f in files if f not in filtered]
if diff:
    ... your dialog code
files = filtered
```

Reply:
I like that; I will do that.